internal/handlers/handlers.go

```diff
@@ -1 +1,54 @@
 package handlers
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w, r, "index.html")
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseMultipartForm(10 << 20); err != nil {
+		http.Error(w, "Ошибка парсинга формы", http.StatusInternalServerError)
```

Comment:
Это проблема клиента
```suggestion
		http.Error(w, "Ошибка парсинга формы", http.StatusBadRequest)
```